pkgs/development/python-modules/langgraph-checkpoint/default.nix

```diff
@@ -4,15 +4,17 @@
   dataclasses-json,
   fetchFromGitHub,
   langchain-core,
+  langgraph-sdk,
   poetry-core,
   pytest-asyncio,
   pytestCheckHook,
   pythonOlder,
+  writeScript,
```

Comment:
I have the feeling that `writeScript` is not used here.

Reply:
Just a hunch, eh? Thank you.

Reply:
Looks like this one found a new way in !

Reply:
How on earth?! 

Fixed.